src/tests/end-to-end/tests/common/settings-drop-down.test.ts

```diff
@@ -3,57 +3,98 @@
 import { Browser } from '../../common/browser';
 import { launchBrowser } from '../../common/browser-factory';
 import { CommonSelectors } from '../../common/element-identifiers/common-selectors';
+import { enableHighContrast } from '../../common/enable-high-contrast';
 import { Page } from '../../common/page';
 import { scanForAccessibilityIssues } from '../../common/scan-for-accessibility-issues';
 
 describe('Settings Dropdown', () => {
-    let browser: Browser;
-    let targetTabId: number;
+    describe('Normal mode', () => {
```

Comment:
Do we really need to wrap them in a "Settings Dropdown" describe block? Just curious to know what it offers!

Reply:
it just wraps that all the tests belongs to settings dropdown